contracts/finance/PaymentSplitter.sol

```diff
@@ -184,6 +184,16 @@ contract PaymentSplitter is Context {
         emit ERC20PaymentReleased(token, account, payment);
     }
 
+    /**
+     *
+     * @dev getter for payees length.
+     *
+     */
+
+    function getPayeesLength() public view returns (uint256) {
```

Comment:
I think it makes sense to call it `totalPayees` or just `payees` to follow the contract convention. I'd wait for comments from @Amxx

Reply:
`totalPayees()` or `payeeCount()` would either be fine by me.

Reply:
Also, this function should move to line 123, just after `payee(uint256)`

Reply:
I went for `totalPayees` and put the function after `totalReleased` following the order of `total*` getters followed by their `*` counterparts in the same order.

Reply:
it has nothing to do with `totalReleased`. This function works in pair with `function payee(uint256)`, so they should go together.